docs/reference/release-notes/6.6.asciidoc

```diff
@@ -1,52 +1,646 @@
-////
-// To add a release, copy and paste the following text,  uncomment the relevant
-// sections, and add a link to the new section in the list of releases in
-// ../release-notes.asciidoc. Note that release subheads must be floated and
-// sections cannot be empty.
-// TEMPLATE
 
-// [[release-notes-n.n.n]]
-// == {es} version n.n.n
+[[release-notes-6.6.0]]
+== 6.6.0 Release Notes
 
-// coming[n.n.n]
+coming[6.6.0]
 
-// Also see <<breaking-changes-n.n>>.
+Also see <<breaking-changes-6.6>>.
 
-// [float]
-// [[breaking-n.n.n]]
-// === Breaking Changes
+[[breaking-6.6.0]]
+[float]
+=== Breaking changes
 
-// [float]
-// [[breaking-java-n.n.n]]
-// === Breaking Java Changes
+Authentication::
+* Enhance Invalidate Token API {pull}35388[#35388] (issues: {issue}34556[#34556], {issue}35115[#35115])
```

Comment:
This item should be removed per discussion with @jaymode (the "breaking" label was removed just a few moments ago, likely after you ran this script)

Reply:
got it, I mis-interpreted @lcawl's comment. I've added it to the enhancements section in eea93cc77d4f01bfbd0131b9c3557925523bd025